lib/src/response/response.rs

```diff
@@ -54,7 +54,7 @@ impl<T> Body<T> {
 }
 
 impl<T: io::Read> Body<T> {
-    /// Attepts to read `self` into a `String` and returns it. If reading or
+    /// Attempts to read `self` into a `String` and returns it. If reading or
     /// conversion fails, returns `None`.
     pub fn into_string(self) -> Option<String> {
         let (mut body, mut string) = match self {
```

Comment:
If you're going to add `into_bytes()`, you should just call `into_bytes().map(String::from_utf8)` here.